File: Website/src/app/shared/ThanhVienThongBao.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ThanhVienThongBao } from 'src/app/shared/ThanhVienThongBao.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ThanhVienThongBaoService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ThanhVienThongBao";
    }
}

File: Service/Implement/ThanhVienThongBaoService.cs
namespace Service.Implement
{
    public class ThanhVienThongBaoService : BaseService<ThanhVienThongBao, IThanhVienThongBaoRepository>
    , IThanhVienThongBaoService
    {
    private readonly IThanhVienThongBaoRepository _ThanhVienThongBaoRepository;
    public ThanhVienThongBaoService(IThanhVienThongBaoRepository ThanhVienThongBaoRepository) : base(ThanhVienThongBaoRepository)
    {
    _ThanhVienThongBaoRepository = ThanhVienThongBaoRepository;
    }
    }
    }

